### DISCUSSION TITLE
Courseplay auf Nitrado Funktioniert nicht

### DISCUSSION THREAD

**Discussion by Play4Hype** [Courseplay/Courseplay_FS22#3209] [Q&A] [answered]
Hallo Farming Freunde,
Ich habe heute mit meinen Freunden einen Nitrado Server aufgesetzt.
Wir wollen unteranderem mit Courseplay spielen aber nach dem Herunterladen gab es einen Fehler weswegen wir den Server zwar starten können aber er nicht auf der Serverliste angezeigt wird.
Laut Fehler überschreitet Courseplay mit 293 Dateien die maximal erlaubte Anzahl von 128 Dateien.

Woran liegt das und kann man das irgendwie fixen?
Vielen Dank schonmal im Voraus. 

![Unbenannt](https://github.com/Courseplay/Courseplay_FS22/assets/168014341/2168b131-0f95-4016-bfe8-19c7119df128)

**Comment by Tensuko** [comment, marked as answer]
Das ist ein Fehler (Warning) der ignoriert werden kann.
Das ist nicht der Grund warum er nicht auftaucht.
Es kann eher daran liegen, dass ihr nicht die exakt gleiche .zip habt und den Filter auf gleiche Mods gestellt habt.

Aber CP läuft problemlos auf allen Servern.

**Reply by Play4Hype** [reply]
Vielen Dank,
Daran haben wir gar nicht gedacht.
Es hat geholfen und jetzt funktioniert es Einwandfrei.
Vielen Dank und Schönen Abend noch.

**Reply by Tensuko** [reply]
Super, dann viel Spaß mit CP :)
